utils/codepointtrie/benches/iai_cpt.rs

```diff
@@ -12,19 +12,19 @@ const SAMPLE_STRING_LATIN1: &str = "Declaration loremips umdolo loremipsompi";
 const SAMPLE_STRING_MIXED: &str = "Dèclaråcion ЗАГАЛЬНА 世界人权宣言 𑄟𑄚𑄬𑄭𑄃𑄇𑄴𑄇𑄥𑄧𑄁𑄢𑄴";
 
 fn get_trie_small() -> CodePointTrie<'static, u8> {
-    CodePointTrie::try_new(
+    CodePointTrie::try_new_with_error_at_end(
```

Comment:
Issue: This is an `iai` bench and we do not want to allocate in it.

I suggest changing `tries::gc_small` to have the correct expanded form.

Basically, try to reduce the number of call sites of `try_new_with_error_at_end`. The only valid call sites I see are:

- toml.rs
- casemap try_from_icu
- _maybe_ test_util.rs

Reply:
Done. Kept it in test_util because it's reading toml files